### DISCUSSION TITLE
Getting "Cannot find module 'next/dist/next-server/server/node-polyfill-fetch'" when deployed

### DISCUSSION THREAD

**Discussion by olaven** [vercel/next.js#14405] [Help]
Hi! Thanks for maintaining this project 😄 

My Next.js-app throws the following error when I am using [nextjs auth0](https://github.com/auth0/nextjs-auth0): 
`2020-06-21T12:31:15.731Z	undefined	ERROR	Uncaught Exception 	{"errorType":"Runtime.ImportModuleError","errorMessage":"Error: Cannot find module 'next/dist/next-server/server/node-polyfill-fetch'\nRequire stack:\n- /var/task/.next/serverless/pages/api/auth/me.js\n- /var/task/now__launcher.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js","stack":["Runtime.ImportModuleError: Error: Cannot find module 'next/dist/next-server/server/node-polyfill-fetch'","Require stack:","- /var/task/.next/serverless/pages/api/auth/me.js","- /var/task/now__launcher.js","- /var/runtime/UserFunction.js","- /var/runtime/index.js","    at _loadUserApp (/var/runtime/UserFunction.js:100:13)","    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)","    at Object.<anonymous> (/var/runtime/index.js:43:30)","    at Module._compile (internal/modules/cjs/loader.js:1133:30)","    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)","    at Module.load (internal/modules/cjs/loader.js:977:32)","    at Function.Module._load (internal/modules/cjs/loader.js:877:14)","    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)","    at internal/main/run_main_module.js:18:47"]}Unknown application error occurredRuntime.ImportModuleError`

Everything works fine when running locally with `dev` and `build`/`start`. 
I had this issue last week. Then it (confusingly) disappeared when I added `.github` and `.vscode` to `.vercelignore`. However, it reappeared a couple days ago. 

I'm using next `9.4.4`, but I have tested with `9.3` as well. 
Furthermore, I'm running node 12, both locally and on vercel (through `engines` in package.json)

Has anyone experienced something similar? Any help is very much appreciated!

**Comment by paleite** [comment]
There's another issue regarding this here: https://github.com/vercel/next.js/issues/16667

**Reply by olaven** [reply]
Thanks for providing the link! 😄
